assets/pages/regular-contributions/__tests__/__snapshots__/regularContributionsReducersTest.js.snap

```diff
@@ -30,6 +30,7 @@ Object {
   "marketingConsent": Object {
     "confirmOptIn": null,
     "error": false,
+    "loading": false,
```

Comment:
I feel like loading is quite ambiguous here, maybe a better term would be `requestPending`

Reply:
will do